lib/useBattlePass.tsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from "react";
+import { Service } from "../types/Service";
+
+interface BPObject {
+  id?: string;
```

Comment:
Is there a reason everything is an optional?

Reply:
Um kinda because i didnt want to populate my local db so i didnt have all the data yet